ball_collision/scripts/index.js

```diff
@@ -0,0 +1,86 @@
+// TODO: right click to delete balls
+// TODO: click/keyboard handler to increase/decrase ball speed
+
+import { getRandomNumber } from "./utils.js";
+import Ball from "./ball.js";
+
+const BALL_COUNT = 100;
```

Comment:
put these in a separate file by name constant.js so there is single source and the value can be tweaked from one source rather than searching the variable in it's internal implemented file